Bottom: 3b14eaf14414dddb5c8776da1efe086bfd88430b
Top:    5fce1fbe5f664ddeb15b6224042b69cf46c51929
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-24 14:42:21 +0100

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index eb4cac2..73468d2 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2748,7 +2748,12 @@ IsPolygonInPolygon (PolygonTypePtr P1, PolygonTypePtr P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
